<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encadenamiento de reglas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Nodo a dictaminar</string>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QPushButton" name="limpiar_pushButton">
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Hechos</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hecho 2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="h1_comboBox"/>
       </item>
       <item row="7" column="3">
        <widget class="QComboBox" name="h2vf_comboBox"/>
       </item>
       <item row="9" column="3">
        <widget class="QComboBox" name="h4vf_comboBox"/>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="h3_comboBox"/>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Hecho 4</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Hecho 3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QComboBox" name="h1vf_comboBox"/>
       </item>
       <item row="8" column="3">
        <widget class="QComboBox" name="h3vf_comboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Hecho 1</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="h4_comboBox"/>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="h2_comboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Reglas</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>2: Si D, E y F Entonces G</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>3: Si H e I Entonces J</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>4: Si C y G Entonces K</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>1: Si A y B Entonces C</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>5: Si G y J Entonces L</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>6: Si K y L Entonces M</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QPushButton" name="aceptar_pushButton">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="nodo_comboBox"/>
    </item>
    <item row="1" column="3" rowspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Modus ponens</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="mp_graphicsView"/>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Resultado:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="resultadomp_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Modus tollens</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="mt_graphicsView"/>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Resultado:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="resultadomt_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
